examples/conda-forge.yaml

```diff
@@ -1,198 +1,226 @@
 pkg-config:
-  win64: [pkg-config]
+  any: [pkg-config]
```

Comment:
nit: Can we use `conda-forge` or just `conda` over `any`?

Ideally we are trying to keep this file in future easy to be integrated with [REP-1111](https://www.ros.org/reps/rep-0111.html).

And for the subkey, I am thinking we can reuse the `conda` [selectors](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors), for example, `osx`, `win64`, `linux64`, or `armv7l`, etc...

What do you think? 

```yaml
pkg-config:
    conda-forge:
        win64: [pkg-config]
noarch-package:
    conda-forge: [noarch-package]
```